src/cdk/v2/destinations/ortto/procWorkflow.yaml

```diff
@@ -57,6 +58,7 @@ steps:
               const trimmedOrttoAttribute = attribute.orttoAttribute.trim().toLowerCase().replace(new RegExp('\\s+', 'g'),'-');
               commonFields.fields[$.fieldTypeMap[attribute.type]+":cm:"+trimmedOrttoAttribute] = $.originalInput.message.context.traits[attribute.rudderTraits]
             )[]
+      console.log("ABVC",commonFields.fields["phn::phone"])
       commonFields.fields = $.removeUndefinedAndNullValues(commonFields.fields)
```

Comment:
commonFields.fields = $.removeUndefinedAndNullValues(commonFields.fields);

=> is this required as `$.removeUndefinedAndNullValues(commonFields)` should take care of this right?

Reply:
Nope it is not required put this for debugging purpose. Removed

Reply:
`$.removeUndefinedAndNullValues(commonFields)` does not take care of that as it is not a recursive function